examples/demo_workflow/app.py

```diff
@@ -0,0 +1,26 @@
+from dapr.ext.workflow.workflow_runtime_options import WorkflowRuntimeOptions
+from dapr.ext.workflow.workflow_client import WorkflowClient
+from dapr.ext.workflow.dapr_workflow_context import DaprWorkflowContext
+from dapr.ext.workflow.workflow_activity_context import WorkflowActivityContext
+
+def hello_world_wf(ctx: DaprWorkflowContext, input):
+    print(f'{input}')
+    yield ctx.call_activity(hello_act, input='  Boil one cup of water. ')
+    yield ctx.call_activity(hello_act, input='  Add tea leaves to the cup. ')
+    yield ctx.call_activity(hello_act, input='  Add boiled water to the cup. ')
+    yield ctx.call_activity(hello_act, input='  Add sugar and milk to the cup. ')
+
+def hello_act(ctx: WorkflowActivityContext, input):
+    print(f'{input}!', flush=True)
+
+daprOptions = WorkflowRuntimeOptions()
+daprOptions.register_workflow(hello_world_wf)
+daprOptions.register_activity(hello_act)
+daprOptions.run()
+
+client = WorkflowClient("localhost","4001")
```

Comment:
What is this port 4001? Where is this configured? Why this port?

Reply:
@DeepanshuA let's remove the explicit port configuration from this example. The default value of `DAPR_GRPC_PORT` will be what most users want.

Reply:
Updated to use DAPR_GRPC_PORT. If that is not found, then "4001" is used.
Used "4001" due to https://github.com/microsoft/durabletask-python/blob/dcf205831124d82ca3886e5dba26d8c7a9ac8d73/durabletask/internal/shared.py#L18